builds/fabric8-che/plugins/che-plugin-bayesian-lang-server/src/main/java/org/eclipse/che/plugin/languageserver/bayesian/server/launcher/BayesianLanguageServerLauncher.java

```diff
@@ -72,4 +76,19 @@ protected Process startLanguageServerProcess(String projectPath) throws Language
         }
     }
 
+    public LanguageServerDescription getDescription() {
+        return DESCRIPTION;
+    }
+
+    private static LanguageServerDescription createServerDescription() {
+        return new LanguageServerDescription("org.eclipse.che.plugin.bayesian.languageserver", //
+                                             null, //
+                                             Arrays.asList( //
+                                             new DocumentFilter(BayesianLanguageServerModule.LANGUAGE_ID, //
```

Comment:
so here we would have 3 languages ? xml, txt, json ? can language description be merged with existing ones ?